duden/locale/de_DE.po

```diff
@@ -0,0 +1,112 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR ORGANIZATION
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: \n"
+"POT-Creation-Date: 2018-05-18 11:12+0200\n"
+"PO-Revision-Date: 2018-05-18 13:39+0200\n"
+"Last-Translator: \n"
+"Language-Team: \n"
+"Language: de\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Generated-By: pygettext.py 1.5\n"
+"X-Generator: Poedit 1.8.7.1\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+
+#: main.py:79
+msgid "Word type: "
+msgstr "Wortart: "
+
+#: main.py:81
+msgid "Usage: "
+msgstr "Nutzung: "
```

Comment:
This should be `"Gebrauch: "`, see e.g. [Barmherzigkeit](https://www.duden.de/rechtschreibung/Barmherzigkeit)